Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -886,9 +886,6 @@ WebProcessDataStoreParameters WebProcessPool::webProcessDataStoreParameters(WebP
 #endif
 
 #if PLATFORM(IOS_FAMILY)
-    std::optional<SandboxExtension::Handle> containerCachesDirectoryExtensionHandle;
-    if (auto directory = websiteDataStore.resolvedContainerCachesWebContentDirectory(); !directory.isEmpty())
```

Comment:
May also drop `WebsiteDataStore::resolvedContainerCachesWebContentDirectory()` and `WebsiteDataStore::m_resolvedContainerCachesWebContentDirectory`